interface/main/mobile/m_functions.php

```diff
@@ -0,0 +1,800 @@
+<?php
+    /**
+     * interface/main/mobile/m_functions.php
+     *
+     * Common mobile items.
+     *
+     * Copyright (C) 2018 Raymond Magauran <magauran@MedExBank.com>
+     *
+     * LICENSE: This program is free software: you can redistribute it and/or modify
+     *  it under the terms of the GNU Affero General Public License as
+     *  published by the Free Software Foundation, either version 3 of the
+     *  License, or (at your option) any later version.
+     *
+     *  This program is distributed in the hope that it will be useful,
+     *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+     *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+     *  GNU Affero General Public License for more details.
+     *
+     *  You should have received a copy of the GNU Affero General Public License
+     *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+     *
+     * @package OpenEMR
+     * @author Ray Magauran <magauran@MedExBank.com>
+     * @link http://www.open-emr.org
+     * @copyright Copyright (c) 2018 MedEx <magauran@MedExBank.com>
+     * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+     */
+    use OpenEMR\Core\Header;
+    use OpenEMR\Services\VersionService;
+    
+    function common_head()
+{
+    ?>
+        
+    <head>
+            
+        <?php Header::setupHeader([ 'jquery-ui', 'jquery-ui-cupertino' ]); ?>
+    
+        <title><?php echo "OpenEMR ". xlt('Mobile'); ?></title>
+        <meta content="width=device-width,initial-scale=1.0" name="viewport">
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
As an aside, should the 3 lines above basically be in every script (there is a mechanism where I can do that if that makes sense)?